src/main/java/com/tagtraum/perf/gcviewer/view/ModelDetailsPanel.java

```diff
@@ -38,6 +38,10 @@
     private DoubleDataMapModel concurrentGcEventModel;
 
     private DoubleDataMapTable vmOperationTable;
+    
+    /** GC Causes Model and table */
+    private DoubleDataMapModel gcCausesModel;
```

Comment:
Please see my comment in the issue. I don't understand the concept of the "causes", yet and am not sure, if this is the right way to go.

Reply:
I have commented on the issue, will likely scrap causes section all together, please let me know what you think.